backend/src/main/java/codezap/category/controller/SpringDocCategoryController.java

```diff
@@ -32,7 +33,9 @@ public interface SpringDocCategoryController {
             @ErrorCase(description = "카테고리 이름이 255자를 초과한 경우", exampleMessage = "카테고리 이름은 최대 255자까지 입력 가능합니다."),
             @ErrorCase(description = "동일한 이름의 카테고리가 존재하는 경우", exampleMessage = "이름이 Spring 인 카테고리가 이미 존재합니다."),
     })
-    ResponseEntity<Void> createCategory(MemberDto memberDto, CreateCategoryRequest createCategoryRequest);
+    ResponseEntity<CreateCategoryResponse> createCategory(MemberDto memberDto,
+            CreateCategoryRequest createCategoryRequest
+    );
```

Comment:
이제까지 켬미의 리팩토링 스타일을 보면 아래와 같이 수정되어야 할 것 같은데 맞을까요?
```suggestion
    ResponseEntity<CreateCategoryResponse> createCategory(
            MemberDto memberDto,
            CreateCategoryRequest createCategoryRequest
    );
```